clients/admin-ui/src/features/privacy-experience/constants.ts

```diff
@@ -1,6 +1,7 @@
 export const COMPONENT_MAP = new Map([
   ["overlay", "Overlay"],
   ["privacy_center", "Privacy center"],
+  ["tcf_overlay", "TCF overlay"],
```

Comment:
I didn't break the cypress tests with this did I?  I think I see those same failures on main?

Reply:
yup no worries—[we know](https://ethyca.slack.com/archives/C05HUQ3J3BM/p1698339737486529) about the failures on main and are working on it separately! thanks for updating this 👍